opslevel/datasource_opslevel_team.go

```diff
@@ -87,42 +126,12 @@ func (teamDataSource *TeamDataSource) Schema(ctx context.Context, req datasource
 		// This description is used by the documentation generator and the language server.
 		MarkdownDescription: "Team data source",
 
-		Attributes: map[string]schema.Attribute{
-			"alias": schema.StringAttribute{
-				MarkdownDescription: "The alias attached to the Team.",
-				Computed:            true,
-				Optional:            true,
-			},
-			"id": schema.StringAttribute{
-				Description: "The ID of this Team.",
-				Computed:    true,
-				Optional:    true,
-			},
-			"name": schema.StringAttribute{
-				Description: "The name of the Team.",
-				Computed:    true,
-			},
-			"members": schema.ListNestedAttribute{
-				NestedObject: schema.NestedAttributeObject{
-					Attributes: memberNestedSchemaAttrs,
-				},
-				Description: "List of team members on the team with email address and role.",
-				Computed:    true,
-			},
-			"parent_alias": schema.StringAttribute{
-				Description: "The alias of the parent team.",
-				Computed:    true,
-			},
-			"parent_id": schema.StringAttribute{
-				Description: "The id of the parent team.",
-				Computed:    true,
-			},
-		},
+		Attributes: teamAttributes(map[string]schema.Attribute{}),
```

Comment:
Looks to me like `"members"`, `"parent_alias"`, and `"parent_id"` would fit here inside `map[string]schema.Attribute{ ... }`